package-lock.json

```diff
@@ -118,29 +111,27 @@
     },
     "node_modules/@babel/compat-data": {
       "version": "7.23.5",
-      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.23.5.tgz",
-      "integrity": "sha512-uU27kfDRlhfKl+w1U6vp16IuvSLtjAxdArVXPa9BvLkrr7CYIsxH5adpHObeAGY/41+syctUWOZ140a2Rvkgjw==",
       "dev": true,
+      "license": "MIT",
```

Comment:
this seems to happen a lot - one command will add a license and lose the resolved / integrity, and npm update will put those back. Do you know why? Or what command did you run that did this?

Reply:
I ran `npm install`. I get a fairly large diff when I do this on main too
```
rm -rf node_modules
rm -rf package-lock.json
npm install
```

I'm a little tempted to just check that in as a separate change to try and get some more reasonable diffs here

Reply:
Ok yeah I split out the first commit which is just doing those steps off of main (which is a pretty large change, so I suspect we're not keeping our package-lock.json up to date properly when updating things). And I re-did these steps when adding the @types/jest dependency.

Reply:
When I do `rm -rf node_modules/ package-lock.json ; npm i; npm i ./packages/convex-helpers` I get a different result. I'll push those result to main. not sure if there's a better way to keep the relative package up to date. I had trouble getting it to update in the package lock without explicitly re-importing it after updating versions